### PR TITLE
[Task] Define common heading class

### PR DESCRIPTION
**Pull request by roseaugusto**
## Related Links

- Task Link: https://www.notion.so/Define-common-heading-class-0037d5fcd94c45bda1eb4ee69aa1be4e
- Route Link: /

## Description

- Make a custom css/config that customize all common headings

## Notes

N/A

## Screenshots

N/A

### CODE REVIEW

File: src/assets/css/button.css [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (charlotteds) [inline src/assets/css/button.css]: Delete unused file.

File: src/pages/components/feature/FeatureSection.js [hunk 2]
```diff
@@ -6,10 +6,8 @@ const FeatureSection = () => {
   return (
     <section>
       {/* Heading */}
-      <div className="w-2/3 lg:w-2/5 mx-auto sm:text-[40px] pt-[60px] sm:pt-[160px]">
-        <h1 className="text-5xl text-center font-semibold text-navy-900">
-          Adopting a pet has never been easier
-        </h1>
+      <div className="w-11/12 md:w-2/3 lg:w-2/5 mx-auto sm:text-[40px] pt-[60px] sm:pt-[160px]">
```
Comment (charlotteds) [inline src/pages/components/feature/FeatureSection.js]: Classes here are also shared by testimonial section heading.

File: src/pages/components/testimonial/TestimonialCard.js [hunk 3]
```diff
@@ -11,12 +11,8 @@ const TestimonialCard = ({ item }) => {
...
       {/* Content */}
       <figcaption className="absolute right-0 bottom-5 mx-5 pb-3 bg-gradient-to-t from-black rounded-lg text-white">
-        <p className="mb-3 mx-5 text-base font-semibold ">
-          {item.text}
-        </p>
-        <p className="mx-5 font-allison text-[2em]">
-          {item.author}
-        </p>
+        <p className="mb-3 mx-5 font-semibold">{item.text}</p>
+        <p className="mx-5 font-allison">{item.author}</p>
```
Comment (charlotteds) [inline src/pages/components/testimonial/TestimonialCard.js]: Please return `text-[2em]` or another class to make the text bigger/clearer.

Comment (roseaugusto) [reply src/pages/components/testimonial/TestimonialCard.js]: @CharlotteDS ohhh haha forgot to put it back, I was trying something earlier

**Review by ume45876139 — APPROVED** [review]
# Working
## Chrome
<details>
<summary>PC</summary>

https://user-images.githubusercontent.com/39412730/165705373-1f0b3150-1a09-409e-be81-391d7150b5b7.mov


</details>

<details>
<summary>Tablet</summary>

https://user-images.githubusercontent.com/39412730/165705428-e14fb2a2-a1ca-464c-80b5-ba3b42407f4e.mp4

</details>

<details>
<summary>Mobile</summary>

https://user-images.githubusercontent.com/39412730/165705342-fc2623d1-b903-4f0b-b9d4-3c9ce080590e.MP4

</details>

- Functionalities are working
- Design is no deformation

# Testing data
test device: Chrome, Safari
test time: 0.5h
comment: 0
iteration: 0
total test time: 0.5h

**Review by charlotteds — APPROVED** [review]

**Pull request merged**